packages/framework/src/services/kafka/kafka-broker.client.test.ts

```diff
@@ -0,0 +1,89 @@
+import { Kafka } from 'kafkajs';
+import * as sinon from 'sinon';
+import { suite } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { KafkaBrokerClient, KafkaConsumePayload } from './kafka-broker.client';
+
+const test = suite('Kafka Broker Client');
+
+test.before.each(async (context) => {
```

Comment:
If we call `restore` in afterEach we should convert this to a beforeAll. Alternatively, we can leave `createSandbox` in beforeEach but would be good to remove the `restore` call from afterEach (context.sandbox is reassigned in any case so clearing it before would be pointless)
```suggestion
test.before((context) => {
```